wire-ios-data-model/Source/Model/User/TypeOfUser.swift

```diff
@@ -16,12 +16,9 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
 
-@class MockUser;
-@class ZMTFailureRecorder;
-#import "ZMUser.h"
-
-@interface ZMUser (Testing)
-
-- (void)assertMatchesUser:(MockUser *)user failureRecorder:(ZMTFailureRecorder *)failureRecorder;
-
-@end
+@objc(ZMTypeOfUser)
+public enum TypeOfUser: Int16, CaseIterable, Sendable {
```

Comment:
question: why not UserType?

2) what's the difference between app and bot?

Reply:
1. Because we have a protocol `UserType` already and it's heavily used (~600 times).
2. regular (default) for regular users, bot for old-style bots (services), app for apps.